src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -157,7 +159,24 @@ def export_config(cmd,
 
     if not skip_features:
         # Get all Feature flags with matching label
-        if destination in ('file', 'appconfig'):
+        if destination == 'file':
+            if format_ == 'properties':
+                skip_features = True
+            else:
+                if not naming_convention:
+                    logger.debug("--naming-convention was not provided. Reserved keywords for feature management section will default to PascalCase.")
+                    naming_convention = 'pascal'
```

Comment:
You can give naming _convention default value in your function with naming_comvention="pascal" in https://github.com/Azure/azure-cli/pull/11637/files#diff-d8019349694cb9d60a9085f6de410f4dR146.
and the default value will show up in command line to users.

Reply:
done

---
In reply to: [361833228](https://github.com/Azure/azure-cli/pull/11637#discussion_r361833228) [](ancestors = 361833228)